packages/presentation/lib/screen/home/bloc/home_bloc.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:math';
+
+import 'package:domain/usecase/generate_number_usecase.dart';
+import 'package:equatable/equatable.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:domain/usecase/check_number_usecase.dart';
+
+part 'home_event.dart';
+
+part 'home_state.dart';
+
+class HomeBloc extends Bloc<HomeEvent, HomeState> {
+  final blocCheckUseCase = CheckNumberUseCase();
+  final blocGenerateUseCase = GenerateNumberUseCase();
+  final Random random = Random();
+
+  HomeBloc() : super(const HomeState()) {
+    on<GenerateNumberEvent>(_generate);
+    on<SubmitNumberEvent>(_submit);
+    on<ChangeSubmitButtonStateEvent>(_changeSubmitButton);
+  }
+
+  void _generate(
+    GenerateNumberEvent event,
+    Emitter<HomeState> emit,
+  ) {
+    final generatedNumber = blocGenerateUseCase.call();
```

Comment:
just use blocGenerateUseCase()

Reply:
disagree

Reply:
why?

Reply:
corrected

Reply:
I can use both ways. No difference between them. Just prefer this one.